index.js

```diff
@@ -51,7 +51,7 @@ function paginate(query, options, callback) {
     }
     promises = {
       docs: docsQuery.exec(),
-      count: this.count(query).exec()
```

Comment:
When using countable querying you should use **@estimatedDocumentCount** as discussed in:  https://github.com/Automattic/mongoose/issues/6713.